packages/slice-machine/components/Simulator/components/SetupModal/steps/excerpts.ts

```diff
@@ -0,0 +1,31 @@
+export type Excerpt = {
+  title: string;
+  excerpt: string;
+};
+
+export const InstallExcerpt = {
+  title: "Install the simulator",
+  excerpt: "Run this command to install the simulator package via npm.",
```

Comment:
should we stick with NPM or offer more like do we have the info if the userr is using NPM or YARN in the frontend ?

Reply:
Good question, I'll ask the PM!

Reply:
Done, code updated